### DISCUSSION TITLE
Support file:/// link in dashboard for click

### DISCUSSION THREAD

**Discussion by jet10000** [meilisearch/product#466] [Feedback & Feature Proposal]
Support click file:/// open like url

file:///Users/mac/github/mlsearch/jpg/34.jpg

**Comment by gmourier** [comment]
Hi @jet10000 👋

Are you talking about the mini-dashboard?

I imagine that if a URL is contained in a document, you would like a link to be automatically affixed to it when it is displayed?

Thanks!

**Reply by jet10000** [reply]
Yes, I want to display local images in mini-dashboard

**Reply by curquiza** [reply]
I ping @mdubus since it's related to the [mini-dashboard](https://github.com/meilisearch/mini-dashboard) if I'm correct

**Reply by mdubus** [reply]
Thanks a lot @curquiza, we already have an issue about this here: https://github.com/meilisearch/mini-dashboard/issues/272
For now, I didn't have time to work on it and find a solution, but I'll try to do it whenever it's possible :)
